themes/bootstrap3/js/facets.js

```diff
@@ -116,18 +116,287 @@ VuFind.register('facetList', function FacetList() {
   return { setup: setup, getContent: getContent, updateContent: updateContent };
 });
 
+/* --- Multi Facets Handling --- */
+VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
+  let globalAddedParams = [];
+  let globalRemovedParams = [];
+  let initialRawParams = window.location.search.substring(1).split('&');
+  let initialFilteredParams = initialRawParams.filter(function isFilter(obj) {
+    return obj.startsWith(encodeURI('filter[]='));
+  });
+  let dateSelectorIds = [];
+  let isMultiFacetsSelectionActivated = false;
+  let callbackOnApply;
+  let callbackWhenDeactivated;
+  let defaultContext;
+
+  function stickApplyFiltersButtonAtTopWhenScrolling() {
+    let applyFilters = defaultContext.getElementsByClassName('apply-filters')[0];
+    let checkbox = defaultContext.getElementsByClassName('apply-filters-selection')[0];
+    window.onscroll = function fixButton() {
+      // To handle delayed loading elements changing the elements offset in the page
+      // We update the offset, depending on if we past the button or not
+      if (checkbox.getBoundingClientRect().bottom < 0) {
+        applyFilters.classList.add('fixed');
+      } else {
+        applyFilters.classList.remove('fixed');
+      }
+    };
+  }
+
+  function updateInitialParams(field, value) {
+    let count = initialRawParams.length;
+    for (let i = 0; i < count; i++) {
+      if (initialRawParams[i].startsWith(field + '=')) {
+        initialRawParams[i] = encodeURI(field + '=' + value);
+        break;
+      }
+    }
+  }
+
+  function handleDateSelector() {
+    for (let dateSelectorId of dateSelectorIds) {
+      let dateParams = [];
+      let allEmptyDateParams = true;
+      let form = document.querySelector('form#' + dateSelectorId);
+      let inputs = form.querySelectorAll('.date-fields input');
+      for (let input of inputs) {
+        if (window.location.search.match(input.name)) {
+          // If the parameter is already present we update it
+          updateInitialParams(input.name, input.value);
+        } else {
+          dateParams.push(encodeURI(input.name + '=' + input.value));
+        }
+        if (input.value !== '') {
+          allEmptyDateParams = false;
+        }
+      }
+      // If at least one parameter is not null we continue the routine for the final URL
+      if (allEmptyDateParams === false) {
+        globalAddedParams = globalAddedParams.concat(dateParams);
+        let inputs = form.querySelectorAll('input');
```

Comment:
Looks like the style checker doesn't like reuse of this variable name. Can you change it to get the build passing? I'll test again once we have our green checkmark back. :-)